src/main/resources/static/js/subscription.js

```diff
@@ -585,7 +518,7 @@ jQuery(document).ready(function () {
 
     function getTemplate() {
         var req = new XMLHttpRequest();
-        req.open("GET", frontendServiceUrl + '/download/subscriptionsTemplate', true);
+        req.open("GET", getFrontEndServiceUrl() + '/download/subscriptionsTemplate', true);
```

Comment:
What is "true" in this case? I think you should create a variable above so that it's easy to see what true in this case actually means :)

Reply:
it is a "get" not "is" so it should return a variable that is not a Boolean.

Reply:
Yes, but i don't know what is sent in to that method without reading the docs for req.open. I think you should create the variable above with a describing name so it's easy to know whats happening.

Reply:
there is a function called forlatUrl, this validates and builds the correct url already before it is stored in the session storage. This could of course be extended to better validation.